platform/android/CHANGELOG.md

```diff
@@ -1,6 +1,42 @@
 # Changelog for the Mapbox Maps SDK for Android
 
-MapLibre welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/maplibre/maplibre-gl-native/blob/master/CONTRIBUTING.md) to get started.
+MapLibre welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/maplibre/maplibre-gl-native/blob/main/CONTRIBUTING.md) to get started.
+
+## main
+
+* Add your pull request...
+* Add missing header guards ((#543)[https://github.com/maplibre/maplibre-gl-native/pull/543])
+* Bump semver from 7.3.7 to 7.3.8 in /platform/android ((#530)[https://github.com/maplibre/maplibre-gl-native/pull/530])
+* Removing unused versions sdk ((#515)[https://github.com/maplibre/maplibre-gl-native/pull/515])
+* Fixes potential NaN when calling `NativeMapView::nativeMoveBy` ((#501)[https://github.com/maplibre/maplibre-gl-native/pull/501])
+* Upgrade Gradle from 3.6.3 to 3.6.4 ((#456)[https://github.com/maplibre/maplibre-gl-native/pull/456])
+* (tag: node-v5.0.1-pre.0) Upgrade libs and remove Jetifier ((#218)[https://github.com/maplibre/maplibre-gl-native/pull/218])
+* Fix android ci workflows ((#476)[https://github.com/maplibre/maplibre-gl-native/pull/476])
+* Bump to JDK 11 in android CI and generate Gradle Wrapper ((#474)[https://github.com/maplibre/maplibre-gl-native/pull/474])
+* Bump ejs from 3.1.7 to 3.1.8 in /platform/android ((#470)[https://github.com/maplibre/maplibre-gl-native/pull/470])
+* Bump semver from 5.7.1 to 7.3.7 in /platform/android ((#461)[https://github.com/maplibre/maplibre-gl-native/pull/461])
+* Bump pixelmatch from 4.0.2 to 5.3.0 in /platform/android ((#460)[https://github.com/maplibre/maplibre-gl-native/pull/460])
+* Bump esm from 3.1.0 to 3.2.25 in /platform/android ((#463)[https://github.com/maplibre/maplibre-gl-native/pull/463])
```

Comment:
Should we include these? I mean it doesn't hurt, but it's not really relevant for the generated code as these are js dependencies?

Reply:
We could put them in their own section...